genestack_client/utils.py

```diff
@@ -137,6 +142,52 @@ def ask_confirmation(question, default=None):
         print 'Unexpected response please input "y[es]" or "n[o]"'
 
 
+def interactive_select(items, message, to_string=None, selected=None):
+    """
+    Asks user to choose one of the items.
+    This function will not end until user input correct answer.
+    This function does not check if the file is connected to a tty device.
+
```

Comment:
I have updated docstring.

Code below is not changed, only moved.

Reply:
The same as above
```
Will be looping until user provides correct credentials.
Does not check if program is attached to tty device.
```